src/applications/disability-benefits/all-claims/content/form0781.jsx

```diff
@@ -1 +1,20 @@
 export const additionalFormsTitle = 'Additional Forms';
+
+// Workflow Choice Page
+export const workflowChoicePageTitle =
+  'Statement about mental health conditions (VA Form 21-0781)';
+
+export const workflowChoicePageDescription =
+  'Some stuff about maiking a selection la la la la la la ';
+
+export const form0781WorkflowChoices = {
+  COMPLETE_ONLINE_FORM: 'optForOnlineForm0781',
+  SUBMIT_PAPER_FORM: 'optForPaperForm0781Upload',
+  OPT_OUT_OF_FORM0781: 'optOutOfForm0781',
+};
```

Comment:
I utilized this in https://github.com/department-of-veterans-affairs/vets-website/pull/33741